composer.json

```diff
@@ -14,22 +14,23 @@
         }
     ],
     "require": {
-        "php": "^7.1",
+        "php": "^7.2",
```

Comment:
Is dit enkel omdat 7.1 inmiddels geen support meer heeft, of werkt het/een van de dependencies ook echt niet op 7.1?

Reply:
- [ ] terugzetten naar php 7.1